lib/nexpose/global_settings.rb

```diff
@@ -115,5 +120,22 @@ def add_control_scanning_to_xml(xml, enabled)
                          'enabled' => enabled ? '1' : '0')
       end
     end
+
+    # Internal method for parsing XML for whether asset linking in enabled.
+    def parse_asset_linking_from_xml(xml)
+      enabled = false
```

Comment:
For new installs, the default is true; for upgrades, false.  AFAIK.

Regarding adding the element if it doesn't exist, yeah, I think leaving it out is probably safer.